image_editor/utils/convert_image.py

```diff
@@ -14,19 +14,33 @@ def image_to_numpy_array(image_path: str) -> np.ndarray:
     return img
 
 
-def numpy_array_to_image(image, image_path: str, file_extension=None) -> str:
-    img = Image.fromarray(image)
+def numpy_array_to_image(image_array, image_path: str, file_extension=None) -> str:
+    img = Image.fromarray(image_array)
     dirname, extension = create_new_image_file(image_path, file_extension)
-    
+
     if not os.path.isdir(dirname):
-        os.makedirs( dirname)
-    
+        os.makedirs(dirname)
+
     list_of_files = Path(dirname).glob(f'*.{extension}')
-    seq_no = max([0]+[int(Path(fn).stem.split('_')[1])  for fn in list_of_files if '_' in Path(fn).stem])
-    new_file_path =dirname + os.sep + f'OPIMG_{seq_no + 1}.{extension}'
+    seq_no = max([0] + [int(Path(fn).stem.split('_')[1]) for fn in list_of_files if '_' in Path(fn).stem])
+    new_file_path = dirname + os.sep + f'OPIMG_{seq_no + 1}.{extension}'
     img.save(new_file_path)
     return new_file_path
 
+
+def cv_to_image(image_cv, image_path: str, file_extension=None) -> str:
+    dirname, extension = create_new_image_file(image_path, file_extension)
+
+    if not os.path.isdir(dirname):
+        os.makedirs(dirname)
+
+    list_of_files = Path(dirname).glob(f'*.{extension}')
+    seq_no = max([0] + [int(Path(fn).stem.split('_')[1]) for fn in list_of_files if '_' in Path(fn).stem])
+    new_file_path = dirname + os.sep + f'OPIMG_{seq_no + 1}.{extension}'
```

Comment:
Can this be made common ? as this is used above also https://github.com/DhimanGhosh/Image_Editor/blob/93beb17223855e24a398a1cd4c1edd5ad36d8fb4/image_editor/utils/convert_image.py#L17